va/va.h

```diff
@@ -1836,7 +1836,8 @@ typedef struct _VAEncMiscParameterRateControl
     /* initial QP at I frames */
     uint32_t initial_qp;
     uint32_t min_qp;
-    uint32_t basic_unit_size;
+    /** @deprecated This value is unused. */
+    va_deprecated uint32_t basic_unit_size;
```

Comment:
basic_unit_size was used in https://github.com/freedesktop/pvr-driver (although no one is using this driver now), maybe other HWs can use it.

Reply:
Can you explain what it does so it can be documented, then?  From what I can get from the source code of that driver it appears to be a measurement in blocks (H.264 -> macroblocks) which gets passed into a binary blob to do something unclear.

(That driver doesn't even compile with anything recent - it depends on an early 0.something version VAAPI from before any API compatibility was maintained.)

Reply:
Actually I don't know what it does, I just guess some HWs support basic unit size based rate control so basic_unit_size is required for these HWs.

Reply:
Ok: I've dropped this commit and modified the following one which updates the documentation appropriately.